guides/source/caching_with_rails.md

```diff
@@ -302,7 +302,7 @@ class Product < ApplicationRecord
 end
 ```
 
-NOTE: Notice that in this example we used the `cache_key` method, so the resulting cache key will be something like `products/233-20140225082222765838000/competing_price`. `cache_key` generates a string based on the model's `id` and `updated_at` attributes. This is a common convention and has the benefit of invalidating the cache whenever the product is updated. In general, when you use low-level caching for instance level information, you need to generate a cache key.
+NOTE: Notice that in this example we used the `cache_key` method, so the resulting cache key will be something like `products/233/competing_price`. `cache_key` generates a string based on the model's `id` attribute. In general, when you use low-level caching for instance level information, you need to generate a cache key.
```

Comment:
I think we should update the code to use the two methods that are now using by default to generate the key with the id and timestamp and also update this note to talk about the two methods.